tools/autograd/derivatives.yaml

```diff
@@ -648,10 +648,10 @@
   self: unsqueeze_to(grad, dim, self.sizes())
 
 - name: std(Tensor self, bool unbiased)
-  self: std_backward(grad, self, result, unbiased)
+  self: var_backward(grad / (2 * result), self, unbiased).masked_fill_(result == 0., INFINITY)
```

Comment:
I think I understand your concern: `masked_fill` is one op, and `result == 0.` is another op.

Reply:
Another instance where `torch.where` special case for float zeros may be useful? (although it's hacky) https://github.com/pytorch/pytorch/pull/8945#issuecomment-401211309

Reply:
Yeah, this is really one of those situations where we should just JIT the kernel and everything would fuse just fine...